.github/workflows/ci.yml

```diff
@@ -175,6 +178,22 @@ jobs:
             UNCOVERED_LINES="${UNCOVERED_LINES//$'\r'/'%0D'}"
             echo "::set-output name=uncovered_lines::$UNCOVERED_LINES"
 
+      - name: Azure login
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.LLVM_AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.LLVM_AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.LLVM_AZURE_SUBSCRIPTION_ID }}
```

Comment:
@voltagebots , @jfrank-summit this doesn't work. Assuming the secrets were configured correctly then the command example is wrong. I'm not sure if I can debug this on my own.